x-pack/dockerlogbeat/docs/configuration.asciidoc

```diff
@@ -0,0 +1,358 @@
+[[log-driver-configuration]]
+== {log-driver} configuration options
+
+++++
+<titleabbrev>Configuration options</titleabbrev>
+++++
+
+experimental[]
+
+Use the following options to configure the {log-driver-long}. You can
+pass these options with the `--log-opt` flag when you start a container, or
+you can set them in the `daemon.json` file for all containers. For detailed
+examples, see <<log-driver-usage-examples>>.
+
+// I've included most of the options that are documented for outputs +
+// logging. I've shortened and edited the descriptions because they were a bit
+// unwieldy. Whichever options we decide to keep should get reviewed.
+
+// Not sure yet if we want to provide usage examples inline in
+// this topic or in a separate topic. Depends on how many options we decide to
+// document. The installation section already shows the basic syntax.
+
+[float]
+=== {ecloud} options
+
+// Do the cloud options work? It seems that we should be able to specify these
+// options without having to also specify hosts, but that doesn't work.
+ 
+[options="header"]
+|=====
+|Option  | Description | Default
+
+|`cloud.id`
+|The Cloud ID found in the Elastic Cloud web console. This ID is
+used to resolve the {stack} URLs when connecting to {ess} on {ecloud}.
+|
+
+|`cloud.auth`
+|The username and password combination for connecting to {ess} on {ecloud}. The
+format is `"username:password"`.
+|
+|=====
+
+[float]
+=== {es} output options
+
+// QUESTION: Does the Elastic Log Driver support these options?
```

Comment:
@fearful-symmetry Does the Elastic Log Driver support any of the options listed on lines 48-50 here?

Reply:
OK, I will add the missing options, but leave out SSL.